korge/src/commonMain/kotlin/com/soywiz/korge/view/View.kt

```diff
@@ -372,7 +374,8 @@ abstract class View internal constructor(
     var colorAdd: ColorAdd
         get() = _colorTransform.colorAdd;
         set(v) {
-            _colorTransform.colorAdd = v
+            if (v.rgba != _colorTransform.colorAdd.rgba)
+                _colorTransform.colorAdd = v
             invalidateColorTransform()
```

Comment:
Can I asume you forgot `{ }` to include the invalidateColorTransform as part of the if?